includes/apple-exporter/components/class-component.php

```diff
@@ -384,7 +402,7 @@ public function anchor() {
 	 * @access public
 	 */
 	public function is_anchor_target() {
-		return ! is_null( $this->uid );
+		return '' !== $this->uid;
```

Comment:
would `! empty()` be good? That would check for `''` and `null`

Reply:
Good call, I was using a string before because I was trying to figure out how the variable was changing through the app, but empty is likely a better solution. Will change.